docker/environment.yml

```diff
@@ -209,3 +209,4 @@ dependencies:
     # Needed to run notebook tests.  Missing indirect recursive dependencies
     # somewhere, should not need to manually add it here.
     - pytest-tornasync
+    - urlpath
```

Comment:
A conda version exist here https://anaconda.org/conda-forge/urlpath, we prefer conda to pip.